go-controller/pkg/ovn/pods.go

```diff
@@ -327,7 +335,24 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) error {
 
 	if annotation == nil {
 		gatewayCIDR, _ := util.GetNodeWellKnownAddresses(nodeSubnet)
-		routes, gwIP, err := getRoutesGatewayIP(pod, gatewayCIDR)
+		var namespaceBackoff = wait.Backoff{Duration: 1 * time.Second, Steps: 7, Factor: 1.5, Jitter: 0.1}
```

Comment:
@rcarrillocruz can we put this somewhere common like pkg/ovn/namespace.go perhaps, since it's used both here and in the hybrid overlay code?

Reply:
done